utils/module.yml.j2

```diff
@@ -0,0 +1,15 @@
+name: hal_nordic
```

Comment:
can you add a comment here `# autogenerated, do not edit, use scripts/...`

Reply:
... alternatively, make script smarter by patching module.yml with rendered blobs section only.